### PR TITLE
add a Dockerfile installation

### PR DESCRIPTION
**Pull request by gregwebs**
This seems to work for me on Linux.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# docker build . -t sncli
+# docker run --rm -it -v /tmp:/tmp -v "$HOME/.sncli/:/root/.sncli/" -v "$HOME/.snclirc:/root/.snclirc" sncli
+FROM python:3.8-buster
```
Comment (swalladge) [inline Dockerfile]: Can this be a more generic? `python:3` should be fine, and pull the latest version at the time.

**Comment by gregwebs** [comment]
The use case is to alias this as a single command for running sncli and not having to deal with any installation issues.

File: Dockerfile (hunk 1 above)
Comment (gregwebs) [reply Dockerfile]: With docker I always try to make things more reproducible and not just pull latest versions

Comment (swalladge) [reply Dockerfile]: Ok, fair enough. :+1:

**Pull request merged**